tests/framework/Samplers/Restart/gold/gridSplit/coarse2.csv

```diff
@@ -0,0 +1,16 @@
+x1,x2,ans
+1.0,2.0,1.0
+1.0,2.5,1.0
+1.5,2.0,1.0
+1.5,2.5,1.0
+2.0,2.0,1.0
+2.0,2.5,1.0
+1.0,2.0,1.0
```

Comment:
Why is RAVEN duplicating this entry (and several others) This needs to be figured out before merging this pull request.

Reply:
Ah, I needed to load it as a different dataset, and then use that in the restart.